package.json

```diff
@@ -18,6 +19,9 @@
     "eslint-config"
   ],
   "devDependencies": {
+    "@types/node": "~16.9.1",
+    "@typescript-eslint/eslint-plugin": "~4.31.1",
+    "@typescript-eslint/parser": "~4.31.1",
```

Comment:
Are some of this dependencies also required for the project that uses the config? If so please make sure you document the installation in the readme. ESLint in general seems to go a `install dependencies your own` policy.